kernel_gateway/notebook_http/__init__.py

```diff
@@ -0,0 +1,89 @@
+from ..base.handlers import default_handlers as default_base_handlers
```

Comment:
Copy/paste the Jupyter copyright header to the top of the file from one of the others.